Configure.ps1

```diff
@@ -82,74 +81,75 @@ of the visual studio solution (App.Config and Web.Config) so that they are consi
     # Active Directory Tenant. This is a GUID which represents the "Directory ID" of the AzureAD tenant into which you want to create the apps.
     # Look it up in the Azure portal in the "Properties" of the Azure AD. 
     # IN THE FUTURE: We'd like to be able to get the tenant name from the tenant ID (or pass the tenant name as an argument)
-    $tenantName = $tenantId
+
+    # Login to Azure PowerShell (interactive: you'll need to sign-in with creds enabling your to create apps in the tenant)
+    $creds = Connect-AzureAD -TenantId $tenantId
+     if (!$tenantId)
+     {
+         $tenantId = $creds.Tenant.Id
+     }
+    $tenant = Get-AzureADTenantDetail
+    $tenantName =  $tenant.VerifiedDomains[0].Name
 
     # Variables for the registration of the AAD application for the Web API Service
     $serviceAadAppName = "TodoListService"
     $serviceHomePage = "https://localhost:44321"
-    $serviceAppIdIRI = "https://"+$tenantName+"/"+$serviceAadAppName
+    $serviceAppIdURI = "https://"+$tenantName+"/"+$serviceAadAppName
```

Comment:
In PowerShell, strings in double-quotes `"` are interpolated.

```powershell
$serviceAppIdURI = "https://$tenantName/$serviceAadAppName"
```

If you want to use a block of code, or refer to a variable which uses a colon `:` then you can encompass it with the `$` character. For example:

```powershell
$script:myScriptVar = 3
$otherVar = "My favourite number is: ${script:myScriptVar}."
```